packages/cli/src/databases/repositories/workflowStatistics.repository.ts

```diff
@@ -98,4 +98,23 @@ export class WorkflowStatisticsRepository extends Repository<WorkflowStatistics>
 			throw error;
 		}
 	}
+
+	async queryNumWorkflowsUserHasWithOver5ProdExecs(userId: User['id']): Promise<number> {
```

Comment:
`Over5` does not match the query as it can be equal as well. Maybe we can explain what this does in a docline and shorten the long method name?

```suggestion
	async queryNumWorkflowsUserHasWithOver5ProdExecs(userId: User['id']): Promise<number> {
```

Reply:
> From what I recall @netroy has been wanting to get rid of workflow statistics

I'm primarily trying to get rid of `EventService`, and other code that populates this table.
If there is a need for this data, then we should definitely keep the table, but we need to figure out a better way to populate it.

Reply:
Renamed the method to match the implementation.

We do kinda stop querying the DB once the CTA is shown, but that happens on the FE side. Do you think we should check the condition already now from `execution_entity` table directly, instead of relying on `workflow_statistics`?

Using events would be nice to implement this feature. Ideally I'd like this to be done somewhere outside the app entirely based on telemetry. Since AFAIK this is pretty one-off thing, I didn't want to spend too much time on that.